fdb-record-layer-core/src/main/proto/record_query_plan.proto

```diff
@@ -460,6 +461,11 @@ message PFirstOrDefaultValue {
   optional PValue on_empty_result_value = 2;
 }
 
+message PFirstOrDefaultStreamingValue {
```

Comment:
Three options:
- `FirstOrDefaultStreamingValue` goes away with the suggested changes elsewhere,
- `FirstOrDefaultStreamingValue` properly implements plan serialization together with putting `PFirstOrDefaultStreamingValue` into `PValue`
- `FirstOrDefaultStreamingValue` does not implement plan serialization  at all (but I think the constraint on the types will then fail to serialize, so properly two options only).

Reply:
`PFirstOrDefaultStreamingValue` already implements plan serialization, I forgot adding it as a possibility in `PValue`. The code works because it is only used currently in `DerivationProperty`.
I just added it to `PValue`.

Reply:
Right, but the derivation is then put into a constraint that is then put into the continuation. I am not sure why this worked.